cmd/options.go

```diff
@@ -82,6 +82,7 @@ func optionFlagSet() *pflag.FlagSet {
 	flags.StringSlice("tag", nil, "add a `tag` to be applied to all samples, as `[name]=[value]`")
 	flags.String("console-output", "", "redirects the console logging to the provided output file")
 	flags.Bool("discard-response-bodies", false, "Read but don't process or save HTTP response bodies")
+	flags.StringP("nic", "n", "", "Use given network interface")
```

Comment:
Can you use a `flags.StringSlice()` here? That would allow users to specify multiple NICs by `k6 run --nic nic1 --nic nic2`, and would also spare us the need to do `strings.Split(ifacesList, ",")` in `js/runner.go`.